composer.json

```diff
@@ -14,10 +14,10 @@
     "ext-pdo": "*",
     "keboola/csv": "^1.1",
     "keboola/db-extractor-logger": "^1.0",
-    "keboola/db-extractor-ssh-tunnel": "^1.0",
+    "keboola/db-extractor-ssh-tunnel": "^1.1",
```

Comment:
Proč se to mění?

Reply:
tam jsem taky povyšoval keboola/retry verzi tak jsem to vzal při jednom když to dělá "stejnou" věc